osu.Game/Scoring/Legacy/LegacyScoreEncoder.cs

```diff
@@ -95,7 +95,7 @@ private string replayStringContent
 
                     foreach (var f in score.Replay.Frames.OfType<IConvertibleReplayFrame>().Select(f => f.ToLegacy(beatmap)))
                     {
-                        replayData.Append(FormattableString.Invariant($"{f.Time - lastF.Time}|{f.MouseX ?? 0}|{f.MouseY ?? 0}|{(int)f.ButtonState},"));
+                        replayData.Append(FormattableString.Invariant($"{(int)Math.Round(f.Time - lastF.Time)}|{f.MouseX ?? 0}|{f.MouseY ?? 0}|{(int)f.ButtonState},"));
```

Comment:
Going to affect spectator, but I guess it's a case of "we shouldn't be using legacy formats anyway".

Reply:
It will only affect score export / local storage.